templates/cloudinary_signed_upload.yaml

```diff
@@ -0,0 +1,65 @@
+name: Cloudinary Signed Upload Endpoint
+author: 
+  name: yakirp
+  link: https://github.com/yakirp
+type: webhook
+description: |
+  A webtask that can be used as an endpoint for signed upload calls (from upload widget, Android, iOS,. etc)
+note:
+  title: How to use with Cloudinary Upload Widget
+  content: |
+    This template creates endpoint which generate and return signature of the upload request
```

Comment:
Can we add one or two sentences here describing the high-level scenario the widget is used for? Something along the lines of what you have in the docs, but maybe shortened. 
> Cloudinary's upload widget is a complete, interactive user interface that enables your users to upload files from a variety of sources to your website or application. The widget, requiring just a couple lines of code to integrate, eliminates the need to develop in-house interactive media upload capabilities.

Reply:
@yavorg Sure you can! This is a great idea.

Reply:
@yavorg I added the description, a bit long, no?